### DISCUSSION TITLE
Undocumented feature for formatter

### DISCUSSION THREAD

**Discussion by Giuliano1993** [biomejs/biome#1138] [General]
I open the discussion in here because wasn't sure about the right issue category, so I followed the hint of opening a discussion.

If it wasn't the right place to do it, just let me know so I can do better next time :) 

In formatter documentation, is missing the doc for `bracketSpacing`, `bracketSameLine`, both in the generic formatter section and in the language specific section.

**Comment by ematipico** [comment]
Wanna open a PR? :)

**Reply by Giuliano1993** [reply]
Yes I'd love to! Can I do that directly or should I open an issue and have it assigned first? 😄

**Reply by ematipico** [reply]
Just do it, you can reference this discussion and close it once the PR is merged 🥷

**Comment by Giuliano1993** [comment]
Closing after PR #1140  merged
